.github/workflows/self-test-qa.yaml

```diff
@@ -16,4 +16,4 @@ jobs:
       fast-test-python-versions: '["3.14"]'
       slow-test-platforms: '["ubuntu-latest"]'
       slow-test-python-versions: '["3.14"]'
-      lowest-python-version: "3.0"
+      lowest-python-version: "3.10"
```

Comment:
This was intentional - did it cause a problem?

If so, could we try with a Python version like 3.6?

Reply:
Yes it was intentional, see https://github.com/canonical/starflow/pull/38/commits/7b734a42de21294e825339d58ceccd105fcfde17. I will try with 3.6 but I am not sure to see why we want to try with such an old version.

Reply:
The setup of the venv is not working with python 3.6, see https://github.com/canonical/starflow/actions/runs/12742989960/job/35512084450?pr=38.
According to https://docs.astral.sh/uv/reference/policies/platforms/ the older version supported by uv is python 3.8 (but according to https://github.com/astral-sh/uv/issues/10454 3.7 could also work). I will try with 3.7 but I doubt we can go any further.

Reply:
Let's do 3.8 then - the intent of  having a version other than 3.10 there is to check that we're actually changing that Python version.